great_expectations/profiler/parameter_builder/metric_parameter_builder.py

```diff
@@ -91,4 +99,10 @@ def _build_parameters(
                 "details": None,
             },
         }
-        return build_parameter_container(parameter_values=parameter_values)
+        build_parameter_container(
```

Comment:
I would love to understand this better - so is the return here unnecessary because the ParameterContainer is passed by reference? If so we will need to change the docstring of this method (`_build_parameters`), also to add mention of the `parameter_container` param

Reply:
@anthonyburdi Correct.